<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ShowTime">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>650</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QLabel" name="OS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>T</string>
      </property>
     </widget>
     <widget class="QLabel" name="OS_1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>551</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="OS_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>561</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="OS_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>541</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="CPU">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="CPU_1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>601</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="CPU_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>611</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="OpenTime">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>420</y>
        <width>641</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="RunTime">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>641</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="CPU_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>581</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="CPU_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>571</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QTableWidget" name="ProcessInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="total">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>540</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Search">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>310</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="Pid">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>360</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Pname">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>390</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Pstatus">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Kill">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>460</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SearchInfo">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>260</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Update">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>220</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>内存信息</string>
     </attribute>
     <widget class="QLabel" name="CPU_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="CPU_32">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="MEM_31">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="MEM_32">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>480</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="MEM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="MEM_1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>441</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="MEM_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>471</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="MEM_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>521</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="MEM_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>531</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU利用率:</string>
      </property>
     </widget>
     <widget class="QLabel" name="MEM_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>511</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="MEM_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>380</y>
        <width>551</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>内存利用率:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>交换空间利用率:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>记录曲线</string>
     </attribute>
     <widget class="QChartView" name="View" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>771</width>
        <height>541</height>
       </rect>
      </property>
      <widget class="QLabel" name="Value">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>367</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>说明</string>
     </attribute>
     <widget class="QLabel" name="detail">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>791</width>
        <height>541</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="ShowCPU">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ShowMem">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reboot">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>590</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>重启</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Halt">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>590</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>关机</string>
    </property>
   </widget>
   <widget class="QLabel" name="ShowSwap">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>ShowTime</zorder>
   <zorder>ShowCPU</zorder>
   <zorder>ShowMem</zorder>
   <zorder>Reboot</zorder>
   <zorder>Halt</zorder>
   <zorder>ShowSwap</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
